packages/webapp/src/components/Crop/CropHeader.jsx

```diff
@@ -5,14 +5,34 @@ import { useTranslation } from 'react-i18next';
 import styles from './styles.module.scss';
 import clsx from 'clsx';
 
-function CropHeader({
-  crop_translation_key,
-  crop_variety_name,
-  supplier,
-  crop_variety_photo_url,
-  onBackClick,
-}) {
+function CropHeader({ variety, onBackClick }) {
   const { t } = useTranslation(['translation', 'crop']);
+  const {
+    crop_variety_name,
+    crop_genus,
+    crop_specie,
+    crop_varietal,
+    crop_cultivar,
+    supplier,
+    crop_variety_photo_url,
+    crop_translation_key,
+  } = variety;
+
+  const crop_scientific_name = () => {
+    console.log(crop_genus);
+    const genus = crop_genus
+      ? crop_genus.charAt(0).toUpperCase() + crop_genus.slice(1).toLowerCase()
+      : '';
+    const species = crop_specie ? crop_specie : '';
+    return `${genus} ${species}`;
+  };
+
+  const varietalDetails = () => {
+    const varietal = crop_varietal ? `var. ${crop_varietal.toLowerCase()}` : '';
+    const cultivar = crop_cultivar ? `'${crop_cultivar}'` : '';
+    return `${varietal} ${cultivar}`;
```

Comment:
The changes look good!
It's a small thing, but when `genus` is empty and there's `species`, we will have unnecessary space before `species`. (same for `varietal` and `cultivar`) If it won't happen, please just ignore!